worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -320,6 +323,37 @@ public void butcherBrush(Player player, LocalSession session,
         player.printInfo(TranslatableComponent.of("worldedit.brush.butcher.equip", TextComponent.of((int) radius)));
     }
 
+    @Command(
+        name = "image",
+        desc = "Image brush, uses images as input"
+    )
+    @CommandPermissions("worldedit.brush.image")
+    public void imageBrush(Player player, LocalSession session,
+                           @Arg(desc = "The name of the image")
+                               String imageName,
+                           @Arg(desc = "The size of the brush", def = "5")
+                               double radius,
+                           @Arg(desc = "The intensity of the brush", def = "5")
+                               double intensity,
+                           @Arg(desc = "Whether to fill or clear blocks", def = "true")
+                               boolean fillMode,
+                           @Arg(desc = "Whether to flatten blocks to the given height", def = "false")
+                               boolean flatMode) throws WorldEditException {
+        BufferedImage image = worldEdit.getImageBrushManager().getImage(imageName);
+        if (image == null) {
+            player.printInfo(TranslatableComponent.of("worldedit.brush.image.imageList", TextComponent.of(imageName))); //TODO add message
```

Comment:
What does this line even mean? There's no "listing" going on here that I can see.

Reply:
Thought of sending a list of brushes here, similar to how WG sends a list of flags on failure (https://github.com/EngineHub/WorldGuard/blob/master/worldguard-core/src/main/java/com/sk89q/worldguard/commands/region/RegionCommands.java#L530)
... it is now implemented as I'd have invisioned it (with the same format/alternating colors as the WG flag list), not sure if the callable should be moved to its extra class.